README.md

```diff
@@ -1,67 +1,69 @@
 
 **What if I receive an unsolicited offer of work?**    
-Refer them to us and follow our guidelines on how to communicate with prospective employers. [in preparation, but likely to include a recommendation that all members put a note on their linkedin profiles that refers prospective employers to FAC]
+Refer them to us and follow our guidelines on how to communicate with prospective employers. It may be useful to display some expanatory text on your LinkedIn page or other professional platforms - FAC will provide this (but it is not mandatory).
 
 **Won't that stop me getting the job?**    
-No. It is often hard, especially with a first job, to realise your own value and it is easy to be over-grateful to a prospective employer who is willing to make you a job offer. Any good employer, once they understand how we operate and the quality of our members is likely to be willing to get involved in a programme that is capable of  providing a steady supply of excellent developers. 
+No. It is often hard, especially with a first job, to realise your own value and it is easy to be over-grateful to a prospective employer who is willing to make you a job offer. Any good employer, once they understand how we operate and the quality of our graduates, is likely to be willing to get involved in a programme that is capable of providing a steady supply of excellent developers. 
 
 **What if I receive a job offer from a friend or ex-employer?**    
-Talk to us. In this case, a suggested response might be:  "I would love to work with you. And if you are looking for any other people, I really recommend you contact the organisation I am a member of [insert agreed FAC contact]. I am sure they would be keen to send you more people."
+Talk to us. In this case, a suggested response might be:  *"I would love to work with you. And if you are looking for any other people, I really recommend you contact the organisation I am part of [insert agreed FAC contact]. I am sure they would be keen to send you more people"*.
 
 **What if I want to apply for a job is in an unrelated field?**    
 If the employer is unlikely to hire developers and the job is in a field where recruiters are not usually used, then there is probably no reason to ask them to contact us. If in doubt, ask.
 
 **If I am freelancing, do I have to give a proportion of my earnings to FAC?**    
-Not unless the work came through FAC [?], in which case the proportion returned to FAC will be decided on a project-by-project basis.
+Not unless the work came through FAC, in which case the proportion returned to FAC will be decided on a project-by-project basis.
```

Comment:
Interesting. I guess if your mate's slipping you £100 to do a bit of work on her website then FAC shouldn't have much to say on the matter.

On reflection, I think this is a remnant from the original document which had a broader remit than this one. I will tweak this shortly